e2e-tests/src/config.rs

```diff
@@ -7,6 +7,10 @@ pub struct Config {
     #[clap(long, default_value = "127.0.0.1:9943")]
     pub node: String,
 
+    /// Whether to use `ws` or `wss` protocol
+    #[clap(long)]
+    pub ssl: bool,
```

Comment:
If `--ssl` is mandatory arg, do we need to update any `scripts` or workflows?

Reply:
tests are passing, so it is not mandatory iiuc; if you add `--ssl` you are just passing `true`

Reply:
Oks